PerfDataExtension/PerfDataExtension.csproj

```diff
@@ -0,0 +1,48 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <OutputType>Library</OutputType>
+    <Version>0.1.2</Version>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
+    <Authors>Microsoft</Authors>
+    <Company>Microsoft</Company>
+    <Product>Performance ToolKit</Product>
+    <AssemblyName>Microsoft.Performance.Toolkit.Plugins.PerfDataExtension</AssemblyName>
+    <RootNamespace>Microsoft.Performance.Toolkit.Plugins.PerfDataExtension</RootNamespace>
+    <Title>Microsoft Linux Tracepoints Decode for Microsoft Performance Toolkit</Title>
+    <Description>Microsoft Performance Toolkit support for Linux Tracepoints, including perf.data files, tracefs formats, and EventHeader events.</Description>
+    <PackageId>Microsoft.Performance.Toolkit.Plugins.PerfDataExtension</PackageId>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <RepositoryUrl>https://github.com/microsoft/Microsoft-Performance-Tools-Linux-Android</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/microsoft/Microsoft-Performance-Tools-Linux-Android</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.LinuxTracepoints.Decode" Version="0.1.2.*">

```

Comment:
You will need a GeneratePathProperty="true" here and post-build or other event to copy the Microsoft.LinuxTracepoints.Types.dll into place. I think lib projects don't grab deps by default

Reply:
Also the package seems to be missing required .dll. When I tried running this after fixing the above WPA threw this error: Microsoft.LinuxTracepoints.Types.dll

Reply:
Looks like the other projects are doing a copy-local of dependencies, just for development-time convenience. I'll add that here too. (Doesn't affect the nuget.)

Reply:
It's also for the plugin folder that we provide with a release and the .ptix neither which will work without this